graphql-java-support/src/main/java/com/apollographql/federation/graphqljava/_Any.java

```diff
@@ -21,8 +21,8 @@
 
 import static graphql.schema.GraphQLScalarType.newScalar;
 
-final class _Any {
-    static final String typeName = "_Any";
+public final class _Any {
```

Comment:
out of curiosity, what's the use case for this being public?

Reply:
It seems worth making it reusable outside of the library.

Reply:
It might be necessary in order to provide support for projects that don't use SDL to create their schema. As @pcarrier pointed out [GraphQL Kotlin](https://github.com/ExpediaDotCom/graphql-kotlin) is one such project that seems to be gaining traction.

That said, it does add significant liability. These can't be changed or moved easily if they remain public. It might be good to look at other mechanisms for sharing the functionality here, like [Hooks in GraphQL Kotlin](https://github.com/ExpediaDotCom/graphql-kotlin/wiki/Schema-Generator-Hooks).